src/logjam/framework/timbre.clj

```diff
@@ -0,0 +1,106 @@
+(ns logjam.framework.timbre
+  "Log event capturing implementation for Timbre."
+  (:require
+   [logjam.appender :as appender]
+   [taoensso.timbre :as timbre])
+  (:import
+   (java.util Date)))
+
+(def ^:private log-levels
+  "The Timbre level descriptors."
+  (->> [:trace :debug :info :warn :error :fatal :report]
+       (map-indexed #(assoc {:name %2
+                             :category %2
+                             :object %2}
+                            :weight %1))))
+
+(defn- extract-event-data [{:keys [vargs level msg_ ^Date instant context ^Throwable ?err thread ?file ^String ?ns-str ?line ?column appender-id]}]
```

Comment:
Not sure this level of deconstruction makes sense at this level, but I'm ambivalent towards this.